src/Game/UI/Gumps/Login/LoginGump.cs

```diff
@@ -226,14 +226,23 @@ public LoginGump(LoginScene scene) : base(0, 0)
             offsetX += 7;
 
             // Text Inputs
-            Add(_textboxAccount = new StbTextBox(5, 16, 190, false, hue: 0x034F)
+            // var savedAccounts = ProfileManager.GetAccounts(World.ServerName);
+            var savedAccounts = new string[] { "Hello World", "hello", "hello", "hello", "hello", "hello", "hello", "hello" };
```

Comment:
Those two lines look bogus ;) Probably remains from testing.

I really should start merging my unit-test-branch. Then this can be covered by unit tests as well.

Reply:
removed.